src/connector/src/parser/common.rs

```diff
@@ -41,6 +41,7 @@ fn do_parse_simd_json_value(dtype: &DataType, v: &BorrowedValue<'_>) -> Result<S
         DataType::Int16 => ensure_i16!(v, i16).into(),
         DataType::Int32 => ensure_i32!(v, i32).into(),
         DataType::Int64 => ensure_i64!(v, i64).into(),
+        DataType::Int256 => todo!(),
```

Comment:
do I understand it correctly that if a user defines a source that ingests JSON data and has an int256 column
1. the raw data in JSON format needs to be a string
2. then we cast it into int256 after parsing

I guess similar reasoning as described here: https://github.com/risingwavelabs/risingwave/issues/7443#issuecomment-1385164468?

Reply:
Yes, we store it as a string in JSON. I will address this issue.